.github/workflows/e2eCI.yaml

```diff
@@ -0,0 +1,56 @@
+---
+name: kotary e2e testing
+on: push
+jobs:
+  kind:
+    runs-on: ubuntu-latest
+    name: Test Kotary Operator on Kind cluster. Kube version
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes:
+          - 1.21
+          - 1.23
+          - 1.24
+          - 1.26
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+      
+      - name: Docker Build image
+        run: | 
+          docker build -t "ci/kotary:${{ matrix.kubernetes }}" .
+          docker image ls | grep ci
+
+      - name: Create cluster KinD
+        uses: helm/kind-action@v1.5.0
+        with:
+          config: e2e/KindConfig/kind-cluster-${{ matrix.kubernetes }}.yaml
+
+      - name: testing cluster kinD
+        run: | 
+              kubectl cluster-info --context kind-chart-testing echo " current-context:" $(kubectl config current-context)
```

Comment:
you seme to be using kind-chart-testing or chart-testing... yet:
there is no helm chart testing, and there is no chart-testing name in the config of the cluster.